skyportal/handlers/api/gcn.py

```diff
@@ -1954,3 +1957,58 @@ class Validator(Schema):
         except Exception as e:
             return self.error(f"Error generating summary: {e}")
         return self.success(data=contents)
+
+
+class LocalizationDownloadHandler(BaseHandler):
+    @auth_or_token
+    async def get(self, dateobs, localization_name):
```

Comment:
Please consider using arrow to parse the dateobs and verify its format, as well as a strip() on the localization_name to minimize errors